File: Doc-Scanner-X/Doc-Scanner-X/CustomTableViewCell.m
//
//  CustomTableViewCell.m
//  Doc-Scanner-X
//
//  Created by dynamsoft on 2018/1/12.
//  Copyright © 2018年 com.dynamsoft. All rights reserved.
//

#import "CustomTableViewCell.h"

@implementation CustomTableViewCell

- (void)layoutSubviews {
    [super layoutSubviews];
    self.imageView.frame = CGRectMake(17, 17, 60, 60);
    if (self.imageView.image.size.width > 0) {
        self.textLabel.frame = CGRectMake(87, self.textLabel.frame.origin.y, self.textLabel.frame.size.width, self.textLabel.frame.size.height);
        self.detailTextLabel.frame = CGRectMake(87, self.detailTextLabel.frame.origin.y, self.detailTextLabel.frame.size.width, self.detailTextLabel.frame.size.height);
    }
}

@end

File: DocScannerX/DocScannerX/CustomTableViewCell.swift
//
//  CustomTableViewCell.swift
//  Doc-Scanner-X
//
//  Created by dynamsoft on 2018/1/4.
//  Copyright © 2018年 Dynamsoft. All rights reserved.
//

import UIKit

class CustomTableViewCell: UITableViewCell {
    
    /// Override `layoutSubviews` to customize tableView cell appearce.
    override func layoutSubviews() {
        super.layoutSubviews()
        self.imageView?.frame = CGRect(x: 17, y: 17, width: 60, height: 60)
        if (self.imageView?.image?.size.width)! > 0 {
            self.textLabel!.frame = CGRect(x: 87, y: self.textLabel!.frame.origin.y, width: self.textLabel!.frame.width, height: self.textLabel!.frame.height)
            self.detailTextLabel!.frame = CGRect(x: 87, y: self.detailTextLabel!.frame.origin.y, width: self.detailTextLabel!.frame.width, height: self.detailTextLabel!.frame.height)
        }
    }

}
